website/data/docs-navigation.js

```diff
@@ -91,6 +91,7 @@ export default [
   {
     category: 'k8s',
     content: [
+      'compatibility',
```

Comment:
This should be a subpage under Upgrade to match the top-level consul core docs

Reply:
e.g.
```
  {
    category: 'upgrading',
    content: ['compatibility'],
  },
```

You'll need to move `upgrading.mdx` to `index.mdx`

Reply:
Done!